CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
 * You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
-* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct.html).
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](/code_of_conduct.md).
 ## General Guidelines
-* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
+* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](systers.io).
```

Comment:
Did you missed it ? Website link is fixed by last merged PR & your PR will break it.